src/components/Button/Button.tsx

```diff
@@ -1,16 +1,27 @@
 import React, { ReactNode } from "react";
 import "styled-components/macro";
+/* import styled from "styled-components/macro"; */
 import { Color } from "variables";
+/* import { Background } from "components/Sidebar/components/UserCard/styles"; */
 
 type Props = {
   children: ReactNode;
+  type?: "primary" | void | undefined;
 };
 
 const Button = (props: Props) => {
+  let backgroundColor: string = "";
+
+  if (props.type === "primary") {
+    backgroundColor = Color.DodgerBlue;
+  } else {
+    backgroundColor = Color.LimedSpruce;
+  }
+
```

Comment:
a bit cleaner way would be something like this:
```suggestion
const backgroundColor = rops.type === "primary" ? Color.DodgerBlue : Color.LimedSpruce
```

Reply:
Yes, the ternary is way cleaner.